src/js/components/Layer/StyledLayer.js

```diff
@@ -197,6 +197,20 @@ const KEYFRAMES = {
   },
 };
 
+const getAnimationStyle = (animate, position, full) => {
+  if (animate === false) {
+    return '';
+  }
+
+  return animate === true || animate === undefined
```

Comment:
when animate would be false? Can we always default to true?

```
  const getAnimationStyle = (animate = true, position, full) => {
    if (!animate) {
       return '';
     }

    return (typeof animate === 'string') ? css`
         animation: ${KEYFRAMES[position][full]} ${animate};
       ` : css`
         animation: ${KEYFRAMES[position][full]} 0.2s ease-in-out forwards;
       `
  };
```

Reply:
@alansouzati the issue attached is asking for an option to also disable animation on Layer, so this PR is providing the interface of `<Layer animate={false} ... >`, in this case, animate will be false.